templates/bot/ts/default/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { CardFactory, Attachment } from "botbuilder";
+import { IBotCommand } from "./commands/IBotCommand";
+import { LearnCommand } from "./commands/LearnCommand";
+import { SSOCommand } from "./commands/SSOCommand";
+import { WelcomeCommand } from "./commands/WelcomeCommand";
+const ACData = require("adaptivecards-templating");
+
+export class Utils {
+  // Bind AdaptiveCard with data
+  static renderAdaptiveCard(rawCardTemplate: any, dataObj?: any): Attachment {
+    const cardTemplate = new ACData.Template(rawCardTemplate);
+    const cardWithData = cardTemplate.expand({ $root: dataObj });
+    const card = CardFactory.adaptiveCard(cardWithData);
+    return card;
+  }
+
+  static CommandClasses = [SSOCommand, WelcomeCommand, LearnCommand];
+
+  static async triggerCommand(keyword: string, parameters: any): Promise<any> {
```

Comment:
It looks that each time when calling triggerCommand(), the code will create each command class again and register it again. Is there any way to register the command only when app starts? @timngmsft  how does donebot register commands and trigger command?